recipe/deploy/update_code.php

```diff
@@ -107,6 +108,14 @@
         cd('{{release_path}}');
         run("$git clone -l $bare .");
         run("$git checkout --force $target");
+    } else if (get('update_code_strategy') === 'remote') {
+        cd('{{release_path}}');
+        run("$git clone -l $bare .");
+        run("$git remote set-url origin $repository", ['env' => $env]);
```

Comment:
clone and remote strategies seem really close to each other. Let's combine remote to clone.

Reply:
Changed it, hopefully this is beter.